server/entities/School.ts

```diff
@@ -0,0 +1,29 @@
+import { AfterLoad, Column, Entity, JoinTable, ManyToMany, PrimaryGeneratedColumn } from 'typeorm';
+
+import { User } from './User';
+
+@Entity()
+export class School {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @ManyToMany(type => User)
+    @JoinTable({ name: 'students' })
```

Comment:
name is the name of the table joining this to users. You want this to be something like "school_students"